requirements.txt

```diff
@@ -4,3 +4,4 @@ protobuf
 onnx>=1.2.1
 scikit-learn>=0.19
 onnxconverter-common>=1.5.1
+pandas
```

Comment:
pandas is used only in investigate.py, I would prefer to change the only that refers to it instead of adding a new dependency not really needed.